src/screens/ChannelListScreen/index.tsx

```diff
@@ -78,6 +82,7 @@ const ChannelListScreen = ({route}) => {
       </View>
 
       <FlatList
+        ref={ref}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a reference to the `FlatList` is appropriate for enabling the scroll-to-top feature. However, please make sure that the `ref` is not used elsewhere in the code that could potentially cause conflicts.
```diff
+        ref={ref}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->